File: exercicios/Python/1177/1177.py
v = int(input())
n = []
j = 0
i = 0   

for i in range (1000):
    print('N[{}] = {}'.format(i,j))
       

    j = j + 1
    i = i + 1
    if j == v:
        j = 0
    if i == 1000:
        break

File: exercicios/R/1177/1177.r
input = file('stdin','r')

num = as.integer(readLines(input,n=1))

j = as.integer(0)
i = as.integer(0)

while(i<1000){
    write(sprintf("N[%d] = %d",i,j),'')

    i = i + 1
    j = j + 1

    if(num == j){
        j = 0
    }
}